scripts/data_ingest/main.py

```diff
@@ -0,0 +1,147 @@
+    },
+    "handlers": {
+        "default": {
+            "level": "INFO",
+            "formatter": "standard",
+            "class": "logging.StreamHandler",
+            "stream": "ext://sys.stdout",  # Default is stderr
+        },
+    },
+    "loggers": {
+        "": {  # root logger
+            "handlers": ["default"],
+            "level": "INFO",
+        },
+        "__main__": {  # if __name__ == '__main__'
+            "handlers": ["default"],
+            "level": "DEBUG",
+            "propagate": False,
+        },
+    },
+}
+
+logging.config.dictConfig(DEFAULT_LOGGING)
+logger = logging.getLogger(__file__)
+
+
+def _log_response(response: requests.Response) -> None:
+    if response.status_code >= 400:
+        logger.error(
+            f"There was an error during a request to {response.url}. "
+            f"STATUS: {response.status_code}, BODY:{response.content!r}"
+        )
+
+    logger.debug(f"STATUS: {response.status_code}, BODY:{response.content!r}")
+
+
+def get_admin_token() -> str:
+    """Go through the login flow & create access token for requests."""
+    admin_user = os.getenv(ADMIN_EMAIL_ENV)
+    admin_password = os.getenv(ADMIN_PASSWORD_ENV)
+
+    if admin_user is None or admin_password is None:
+        raise RuntimeError("Admin username & password env vars must be set")
+
+    response = requests.post(
+        get_request_url("/api/tokens"),
+        headers={"Content-Type": "application/x-www-form-urlencoded"},
+        data={"username": admin_user, "password": admin_password},
+    )
+    _log_response(response=response)
+
+    token: str = response.json()["access_token"]
+    return token
+
+
+def get_admin_auth_headers() -> dict[str, str]:
+    """Create the required auth headers for requests."""
+    if (admin_user_token := os.getenv(ADMIN_TOKEN_ENV)) is None:
+        admin_user_token = get_admin_token()
+        os.environ[ADMIN_TOKEN_ENV] = admin_user_token
+
+    return {
+        "Authorization": "Bearer {}".format(admin_user_token),
+        "Accept": "application/json",
+    }
+
+
+def get_request_url(endpoint: str) -> str:
+    api_host = os.getenv("API_HOST", "http://backend:8888").rstrip("/")
+    return f"{api_host}/{endpoint}"
+
+
+def post_data_ingest(ingest_csv_path: Path) -> requests.Response:
+    response = requests.post(
+        get_request_url(BULK_IMPORT_ENDPOINT),
+        headers=get_admin_auth_headers(),
+        files={"law_policy_csv": (None, ingest_csv_path.read_bytes(), "text/csv")}
+    )
+    return response
+
+
+def main(ingest_csv_path: Path):
+    """Initial loader for alpha users.
+
+    Bulk import data into the backend API database from CSV.
+
+    :return: None
+    """
+    try:
+        data_ingest_response = post_data_ingest(ingest_csv_path=ingest_csv_path)
+    except Exception:
+        logger.exception("Calling the endpoint raised an unexpected exception.")
+        sys.exit(1)
+
+    if data_ingest_response.status_code == HTTPStatus.ACCEPTED:
```

Comment:
Do we want to dump out the returning `BulkImportValidatedResult` object?

Reply:
it does that in the info message below, or did you mean to stdout?

Reply:
okay, have checked & this is definitely now working as expected